msteams-platform/m365-apps/combine-office-add-in-and-teams-app.md

```diff
@@ -0,0 +1,621 @@
+
+This article will walk you through the steps to add an Outlook Add-in to a Teams app, test it, then deploy it to Azure.
+
+## Prerequisites 
+
+To configure an Office Add-in, ensure that you meet the following requirements: 
+
+- A Microsoft 365 account that includes Teams to test the application. Alternatively, you can have separate subscriptions to both Microsoft 365 and Teams. For example, a test account with *.onmicrosoft.com through the [Microsoft 365 Developer Program](/office/developer-program/microsoft-365-developer-program). 
+- Your Microsoft 365 account is added as an account in desktop Outlook. For more information, see [add an email account to Outlook](https://support.microsoft.com/office/add-an-email-account-to-outlook-e9da47c4-9b89-4b49-b945-a204aeea6726).
+ - An Azure account with active subscription to deploy the Teams app to Azure. If you don't already have one, you can create your [free Azure account](https://azure.microsoft.com/free/).
+ - A Teams app created using the latest version of Teams Toolkit.
+
+## Overview
+
+The following are the major steps to adding an Outlook Add-in to a Teams app.
+
+1. [Prepare the Teams app project](#prepare-the-teams-app-project)
+1. [Create an Office Add-in project](#create-an-outlook-add-in-project) that is initially separate from your Teams app project.
+1. [Merge the manifest](#merge-the-manifest) from the Outlook Add-in project into the unified manifest for Microsoft 365.
+1. [Copy the Outlook Add-in files to the Teams app project](#copy-the-outlook-add-in-files-to-the-teams-app-project).
+1. [Edit the tooling configuration files](#edit-the-tooling-configuration-files).
+1. [Run the app and add-in locally at the same time](#run-the-app-and-add-in-locally-at-the-same-time)
+1. [Move the application to Azure](#move-the-application-to-azure).
+
+## Prepare the Teams app project
+
+Begin by separating the source code for the tab (or bot) into its own subfolder. These instructions assume that the project initially has the following structure. To create a Teams app project with this structure, be sure you're using the latest released version of Teams Toolkit. 
+
+```
+|-- .vscode/
+|-- appPackage/
+|-- build
+|-- env/
+|-- infra/
+|-- node_modules/
+|-- src/
+|-- .gitignore
+|-- .localConfigs
+|-- .webappignore
+|-- package-lock.json
+|-- package.json
+|-- teamsapp.local.yml
+|-- teamsapp.yml
+|-- tsconfig.json
+|-- web.config
+```
+
+> [!NOTE]
+> If you're working with a new Teams tab project, the node_modules folder and the package-lock.json file isn't present. The node_modules is created in a later step when you run `npm install` in the root of the project. The build folder isn't present unless and until you run a build script on the project. 
+
```

Comment:
Please add a stem sentence to start the steps.

Suggestion:
To separate the source code for the tab or bot, perform the following steps:

Reply:
done